LICENSE

```diff
@@ -1,2 +1,22 @@
-YEAR: 2020-
-COPYRIGHT HOLDER: the SKELETON authors - see file DESCRIPTION
\ No newline at end of file
+YEAR: 2021-
```

Comment:
Is it formally "legal" to change the license to MIT? @cbeleites

Reply:
We are OK on this package, it is entirely new and no material from the legacy version is being used (at least to this point, there will be unit tests and the vig but I'm not sure we would worry about this from a license POV).